### DISCUSSION TITLE
how to cancel epthook

### DISCUSSION THREAD

**Discussion by romanholidaypancakes** [HyperDbg/HyperDbg#226] [Q&A] [answered]
I have an epthook for a function, but after a while I don't need it anymore. How to cancel it, I can't find the relevant command

**Comment by SinaKarvandi** [comment, marked as answer]
Thanks for opening this issue.

This is a question, so it's better to move it in the discussion part.

But, as the answer you should use the 'events' command in HyperDbg. Please visit: 
https://docs.hyperdbg.org/commands/debugging-commands/events
